src/http-gateways/trustless-gateway.md

```diff
@@ -301,6 +302,32 @@ of their presence in the DAG or the value assigned to the "dups" parameter, as
 the raw data is already present in the parent block that links to the identity
 CID.
 
+## CAR `meta` (content type parameter)
+
+The `meta=eof` parameter allows clients to request the server to include additional metadata about the
+CAR to be included at the end of the response body.
```

Comment:
:thought_balloon: 

If this IPIP ends up doing `<CARv1> <0x00 byte> <dag-cbor-metadata>` would it be less mysterious and more self-explanatory if we did `eof=cbor`?